cmd/translateapp/main.go

```diff
@@ -3,13 +3,18 @@ package main
 import (
 	"context"
 	"log"
+	"net/http"
 	"os/signal"
 	"syscall"
+	"translateapp/internal/app/apiserver"
 )
 
 func main() {
 	log.Printf("starting...")
 	ctx, done := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
+
+	srv:= apiserver.NewServer()
+	http.ListenAndServe(":8080",srv)
```

Comment:
There is no graceful shutdown implementation. When application receives SIGINT or SIGTERM it will hang instead of shutdown.